docker/standalone/Dockerfile

```diff
@@ -1,8 +1,3 @@
-FROM gradle:8 as builder
```

Comment:
The jar is compiled on the pipeline at a previous step. This ensures two things:

1. Drastically reduces build time for linux/arm64 images. It was taking aroung 3 to 5 mins just to compile the jar
2. The jar can also be used by the `lh-server` image